azure-pipelines.yml

```diff
@@ -7,26 +7,24 @@ resources:
   containers:
   - container: ubuntu18
     image: ubuntu:18.04
+  - container: wpilib2018
+    image: wpilib/roborio-cross-ubuntu:2018-17.10
+  - container: wpilib2019
+    image: 'wpilib/roborio-cross-ubuntu:2019-18.04'
```

Comment:
why is one image quoted and the other not?

Reply:
Fixed.